core/src/App.html

```diff
@@ -800,16 +800,22 @@
           e.data
         );
         const customMessageListener = customMessagesListeners[message.id];
-        if (typeof customMessageListener === 'function') {
-          const microfrontend = LuigiElements.getMicrofrontends().find(mf =>
-            IframeHelpers.isMessageSource(e, mf.container)
-          );
-
-          customMessageListener(
-            message,
-            microfrontend,
-            GenericHelpers.removeInternalProperties(iframe.luigi.currentNode)
+        if (customMessageListener == undefined) {
```

Comment:
Just being picky: I would love to see the inverted way, since ifs should preferable check the truthy and and most used case first. 
So the `if (typeof customMessageListener === 'function') {` could stay and the else is the warning.